lib/cc/engine/markdownlint.rb

```diff
@@ -81,6 +83,24 @@ def content(code)
           contents[code] = content
         end
       end
+
+      def fingerprint(check_name, path, line_number)
+        md5 = Digest::MD5.new
+        md5 << check_name
+        md5 << path
+        md5 << read_line(path, line_number).gsub(/\s/, "")
+        md5.hexdigest
+      end
+
+      def read_line(path, line_number_to_read)
```

Comment:
Fun to see how these implementations compare between Ruby and Haskell: https://github.com/filib/codeclimate-shellcheck/pull/50/files#diff-3042f35a0ab0bb497d53ab0f16cba408R35